shopfloor_batch_automatic_creation/tests/test_batch_create.py

```diff
@@ -140,24 +144,27 @@ def test_volume(self):
         volume_2 = 0.25
         volume_4 = 0.6
         self.product_a.volume = 0.1
+        cm_uom = self.env.ref("uom.product_uom_cm")
         self.env["product.packaging"].sudo().create(
             {
                 "name": "pair",
                 "product_id": self.product_a.id,
                 "qty": 2,
-                "height": 1000,
-                "width": 1000,
-                "lngth": volume_2 * 1000,
+                "height": 100,
+                "width": 100,
+                "packaging_length": volume_2 * 100,
+                "length_uom_id": cm_uom.id,
```

Comment:
Thanks for this change, and sorry for the bad advice :)
I wasn't aware of the `length_uom_id` available in v14, we'll create our own `mm` UoM locally.

Reply:
Good day @sebalix,
No problem. We are learning everyday. Thanks a lot for your feedback.